### PR TITLE
fix(1302): fixed type conflicts with apps service and lighthouse service

### PR DESCRIPTION
**Pull request by akhilmhdh**
# Fixes

- fix(1302) - Type conflicts with apps service and lighthouse service

# Explain the feature/fix

1. Fixed type conflict due to lighthouse schema with app service schema
2. Made changes to SPA with the service type definition changes
3. Better comments added

## Does this PR introduce a breaking change

No

## Screenshots

NA

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: packages/lighthouse-service/src/property-manager/typedef.graphql [hunk 1]
```diff
@@ -1,102 +1,102 @@
...
+  to get a particular lighthouse property by id
   """
-  fetchProperty(id: ID!): PropertyType
+  getLHPropertyById(id: ID!): LHPropertyType
 }
 
 type Mutation {
   """
-  create a new property
+  create a new lighthouse property
```
Comment (riginoommen) [inline packages/lighthouse-service/src/property-manager/typedef.graphql:79]:
```suggestion
  Create a new lighthouse property
```

File: packages/lighthouse-service/src/property-manager/typedef.graphql [hunk 2]
```diff
@@ -1,102 +1,102 @@
...
   """
-  createProperty(property: AddPropertyInput!): PropertyType
+  createLHProperty(property: AddLHPropertyInput!): LHPropertyType
   """
-  update a property
+  update a lighthouse property
```
Comment (riginoommen) [inline packages/lighthouse-service/src/property-manager/typedef.graphql:83]:
```suggestion
  Update a lighthouse property
```

File: packages/lighthouse-service/src/property-manager/typedef.graphql [hunk 3]
```diff
@@ -1,102 +1,102 @@
...
   """
-  updateProperty(id: ID!, data: UpdatePropertyInput): PropertyType
+  updateLHProperty(id: ID!, data: UpdateLHPropertyInput): LHPropertyType
   """
-  delete a property
+  delete a lighthouse property
```
Comment (riginoommen) [inline packages/lighthouse-service/src/property-manager/typedef.graphql:87]:
```suggestion
  Delete a lighthouse property
```

File: packages/lighthouse-service/src/property-manager/typedef.graphql [hunk 4]
```diff
@@ -1,102 +1,102 @@
...
   """
-  deleteProperty(id: ID!): PropertyType
+  deleteLHProperty(id: ID!): LHPropertyType
   """
-  add an app to a property
+  add an app to a lighthouse property
```
Comment (riginoommen) [inline packages/lighthouse-service/src/property-manager/typedef.graphql:91]:
```suggestion
  Add an app to a lighthouse property
```

File: packages/lighthouse-service/src/property-manager/typedef.graphql [hunk 5]
```diff
@@ -1,102 +1,102 @@
...
   """
-  createApp(propertyId: ID!, appData: AddPropertyAppInput!): PropertyType
+  createLHApp(propertyId: ID!, appData: AddLHPropertyAppInput!): LHPropertyType
   """
-  update an app from a property
+  update an app of a lighthouse property
```
Comment (riginoommen) [inline packages/lighthouse-service/src/property-manager/typedef.graphql:95]:
```suggestion
  Update an app of a lighthouse property
```

File: packages/lighthouse-service/src/property-manager/typedef.graphql [hunk 6]
```diff
@@ -1,102 +1,102 @@
...
   """
-  updateApp(appId: ID!, appData: PropertyAppInput!): PropertyType
+  updateLHApp(appId: ID!, appData: LHPropertyAppInput!): LHPropertyType
   """
-  delete an app from a property
+  delete an app of a lighthouse property
```
Comment (riginoommen) [inline packages/lighthouse-service/src/property-manager/typedef.graphql:99]:
```suggestion
  Delete an app of a lighthouse property
```

**Review by riginoommen — CHANGES_REQUESTED** [review]

File: packages/lighthouse-service/src/property-manager/typedef.graphql (hunk 1 above)
Comment (akhilmhdh) [reply packages/lighthouse-service/src/property-manager/typedef.graphql]: Done

File: packages/lighthouse-service/src/property-manager/typedef.graphql (hunk 5 above)
Comment (akhilmhdh) [reply packages/lighthouse-service/src/property-manager/typedef.graphql]: Done

**Review by riginoommen — APPROVED** [review]

**Review by ghanlohar — APPROVED** [review]

**Pull request merged**